dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/RequestParamUtilsTest.java

```diff
@@ -162,4 +175,40 @@ private TrackedEntityAttribute trackedEntityAttribute(String uid) {
     tea.setUid(uid);
     return tea;
   }
+
+  @Test
+  void shouldFailWhenOrgUnitSuppliedAndOrgUnitModeAccessible() {
+    Exception exception =
+        assertThrows(
+            BadRequestException.class,
+            () -> validateOrgUnitParams(Set.of(UID.of(orgUnit.getUid())), ACCESSIBLE));
+
+    assertStartsWith(
+        "orgUnitMode ACCESSIBLE cannot be used with orgUnits.", exception.getMessage());
+  }
+
+  @Test
+  void shouldFailWhenOrgUnitSuppliedAndOrgUnitModeCapture() {
+    Exception exception =
+        assertThrows(
+            BadRequestException.class,
+            () -> validateOrgUnitParams(Set.of(UID.of(orgUnit.getUid())), CAPTURE));
+
+    assertStartsWith("orgUnitMode CAPTURE cannot be used with orgUnits.", exception.getMessage());
+  }
+
+  @Test
+  void shouldMapOrgUnitModeWhenOrgUnitSuppliedAndOrgUnitModeSelected() {
```

Comment:
maybe use `shouldPass` to oppose the `shouldFail` 😄 

```suggestion
  void shouldPassWhenOrgUnitSuppliedAndOrgUnitModeSelected() {
```